docs/usage/manual_cleanup.md

```diff
@@ -0,0 +1,105 @@
+
+## Cloud provider resource
+
+As a part of Shoot hosting, gardener create different cloud providers resources like vpc, vm, object in objet store, etc. Missing out on cleaning up some of these resource may end up costing you unnecessary. These resources are spread up across max three accounts one for shoot cloud provide account, dns account, and object store account though usually all might be same for development purpose. Please follow the doc to cleanup all cloud provider resources.
+
+### Shoot cloud provider account
+
+Gardener as a part of shoot reconciliation create the different resource in end-users account. i.e the shoot owners account.
+
+#### AWS
+- Delete the VPC with name `ShootTechnicalID`.
+- Delete the IAM roles related to shoot with name:
+    - `ShootTechnicalID-bastions`
+    - `ShootTechnicalID-nodes`
+
+#### Azure
+- Delete the resource group with the name `ShootTechnicalID-ShootUID[:5]`.
+
+#### GCP
+- Delete the VPC network with the name `ShootTechnicalID`.
+
+#### Openstack
+- Delete the network with the name `ShootTechnicalID` and its dependencies.
+
+
+### Cleaning up Backups
+
+Gardener regularly take the backup of etcd backing shoot cluster.
+While purging shoot one has to consider cleaning up these backups as well.
+You can get the backup provider account
+
+#### AWS
+- Go to AWS S3 web-console and login to configured backup provider account. This is the same account which you have configured under `seed.spec.backup` section.
+- Go to the bucket used to shoot backup. You can find the bucket name from the `backupentry.core.gardener.cloud` object with name `ShootTechnicalID--ShootUID` at `.spec.bucketname`.
+- Delete all the objects under prefix `ShootTechnicalID--ShootUid` from that bucket.
+
+#### Azure
+- Go to Microsoft Azure web-console and login to configured backup provider subscription. This is the same account which you have configured under `seed.spec.backup` section.
+- Go to the bucket used for shoot backup. You can find the bucket name from the `backupentry.core.gardener.cloud` object with name `ShootTechnicalID--ShootUID` at `.spec.bucketname`.
+- Delete all the objects under prefix `ShootTechnicalID--ShootUid` from that bucket.
+
+#### GCP
+
+- Go to Google cloud platform web-console and login to configured backup provider account. This is the same account which you have configured under `seed.spec.backup` section.
+- Go to the bucket used for shoot backup. You can find the bucket name from the `backupentry.core.gardener.cloud` object with name `ShootTechnicalID--ShootUID` at `.spec.bucketname`.
+- Delete all the objects under prefix `ShootTechnicalID--ShootUid` from that bucket.
+
+#### Openstack
+
+- Go to Openstack Horizon console and login to configured backup provider account. This is the same account which you have configured under `seed.spec.backup` section.
```

Comment:
Openstack is a bit tricky compared to the other Infrastructure services.
The Horizon is the default dashboard, but it is not mandatory. As the documentation general enough, I would avoid to specify Horizon
```suggestion
- Go to the Openstack Dashboard and login to configured backup provider account. This is the same account which you have configured under `seed.spec.backup` section.
```